readme.md

```diff
@@ -139,7 +167,13 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+function Instructor(name, gift){
+  var.self = this;
+  this.name= name;
+  this.receivePresent = function (gift) {
+    console.log(self.name + "promptly drops the" +gift+ "on the floor.")
+  }
+}
```

Comment:
This is totally valid, though you would save memory by adding the `receivePresent` method to the prototype, rather than each instance of `Instructor`. Also, what about the instantiation step?